packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -375,23 +375,31 @@ class RenderEditable extends RenderBox {
     return _textPainter.maxIntrinsicWidth;
   }
 
-  // This does not required the layout to be updated.
-  double get _preferredLineHeight => _textPainter.preferredLineHeight;
+  /// The height of a typical text line in logical pixels.
+  ///
+  /// This height is based on properties of the text's style such as
+  /// font size etc and could change when those properties change.
```

Comment:
But how would a user of this API know when that happens?

For example, if you change the font-size of a text field that is showing its toolbar, will the toolbar's position be adjusted?